sql-to-dbsp-compiler/SQL-compiler/src/test/java/org/dbsp/sqlCompiler/compiler/sql/MultiCrateTests.java

```diff
@@ -99,6 +100,23 @@ CREATE VIEW v1 AS SELECT COUNT(*) FROM t1
         this.compileMultiCrate(file.getAbsolutePath());
     }
 
+    @Test
+    public void testPackagedDemos() throws SQLException, IOException, InterruptedException {
+        final String projectsDirectory = "../../demo/packaged/sql";
+        final File dir = new File(projectsDirectory);
+        FilenameFilter filter = (_d, name) -> !name.contains("setup") && name.endsWith(".sql");
+        String[] sqlFiles = dir.list(filter);
+        assert sqlFiles != null;
+        for (String sqlFile: sqlFiles) {
+            // System.out.println(sqlFile);
+            String basename = Utilities.getBaseName(sqlFile);
+            String udf = basename + ".udf.rs";
+            File udfFile = new File(dir.getPath() + "/" + udf);
+            if (!udfFile.exists())
+                this.compileMultiCrate(dir.getPath() + "/" + sqlFile);
```

Comment:
Demos with UDFs are not tested?

Reply:
I would have to copy the udf in the right place for that, was lazy